src/Storages/Hive/StorageHive.h

```diff
@@ -88,6 +88,9 @@ class StorageHive final : public shared_ptr_helper<StorageHive>, public IStorage
     HiveFilePtr
     createHiveFileIfNeeded(const FileInfo & file_info, const FieldVector & fields, SelectQueryInfo & query_info, ContextPtr context_);
 
+    bool isColumnOriented();
```

Comment:
I believe the build should fail when you merge with master as this method is not virtual in IStorage. Please merge with master.